### ISSUE TITLE
Annotation upload does not preserve user bounding boxes

### ISSUE THREAD

**Issue by erjel** [scalableminds/webknossos-libs#835]
## Context
- Affected library: webknossos

While uploading annotations, I noticed that user bounding boxes are not preserved over a round-trip via `webknossos.org`.
I.e. the following script `test.py`:
```python
import webknossos as wk

REMOTE_DATASET_URL = 'https://webknossos.org/datasets/Bao_Lab_Sloan_Kettering_Institute/c_elegans_embryo_475min'

WK_USER = '<User Name>'
WK_TOKEN = '<Auth Token>'
WK_URL = 'https://webknossos.org'


ds = wk.Dataset.open_remote(REMOTE_DATASET_URL)

annotation_up = wk.Annotation(
    name = 'test_bbbox_upload',
    dataset_name = ds.name,
    voxel_size = ds.voxel_size,
    owner_name = WK_USER,
)
# Add data to prevent uploading empty content
group = annotation_up.skeleton.add_group("a group")
tree = group.add_tree("a tree")
node_1 = tree.add_node(position=(0, 0, 0), comment="node 1")

# Create a test bounding box
annotation_up.user_bounding_boxes = [wk.BoundingBox.from_tuple6((1024, 512, 128, 64, 64, 64))]

with wk.webknossos_context(url=WK_URL, token=WK_TOKEN):
    annotation_url = annotation_up.upload()
    annotation_down = wk.Annotation.download(annotation_url)

assert annotation_down.user_bounding_boxes == annotation_up.user_bounding_boxes, \
    f'\n{annotation_down.user_bounding_boxes=}' \
    f'\n{annotation_up.user_bounding_boxes=}'


```
results in assertion error:
```Python
Traceback (most recent call last):
  File "test.py", line 27, in <module>
    assert annotation_down.user_bounding_boxes == annotation_up.user_bounding_boxes, \
AssertionError:
annotation_down.user_bounding_boxes=[]
annotation_up.user_bounding_boxes=[BoundingBox(topleft=(1024, 512, 128), size=(64, 64, 64))]
```

## Expected Behavior

I would expect preserved user bounding boxes for a round-trip via `webknossos.org`.

## Current Behavior
Instead I loose the hard-coded bounding boxes.

## Steps to Reproduce the bug
```cmd
conda env create webknossos python=3.9
conda activate webknossos
pip install webknossos
python .\test.py
```

## Your Environment for bug

- Operating System and version: Windows 10
- Version of webKnossos-libs (Release or Commit): webknossos==0.10.25

**Comment by jstriebel** [comment]
@erjel Thank you very much for the bug report. A fix is in progress and will be released soon, I'll let you know when this is the case. In the meantime you can circumvent this problem by explicitly setting `id` of a BoundingBox (currently only possible via the constructor) to a unique string of an integer, e.g.
```
annotation_up.user_bounding_boxes = [wk.BoundingBox((1024, 512, 128), (64, 64, 64), id="1")]
```

**Comment by jstriebel** [comment]
@erjel We just released version `0.11.0` which includes a fix for this. Also, we removed the `id` attribute, since it is not useful for users and just an implementation detail.
https://github.com/scalableminds/webknossos-libs/releases/tag/v0.11.0

I hope this resolves this bug more nicely for you.
